pkg/apis/wavefront/v1alpha1/wavefrontproxy_types.go

```diff
@@ -73,6 +67,14 @@ type WavefrontProxySpec struct {
 	// For pinning Proxy to a specific version, you will need to set this option to false.
 	// We support only minor version Auto Upgrades.
 	EnableAutoUpgrade bool `json:"enableAutoUpgrade,omitempty"`
+
+	// Openshift Specific configurations starts from here
+
+	// Set to true when running proxy in Openshift platform.
+	Openshift bool `json:"openshift,omitempty"`
+
+	// The name of the storage claim to be used for creating proxy buffers directory.
+	StorageClaimName string `json:"storageClaimName,omitempty"`
```

Comment:
is this required only for openshift or for any Proxy versions <= 5.5 in general ?

Reply:
This is for any proxy running on Openshift.

Reply:
ok, could we make a note against the parameter description about the same. Something like "// The name of the storage claim to be used for creating proxy buffers directory. This is applicable only in an openshift environment"

This is to avoid confusion when customer sees the description in CRD.  Customer may think this is required for openshift but can be used in all envrionments (which is not the case based on current code i think)

Reply:
Updated comment